src/main/java/org/springframework/metrics/instrument/scheduling/ExecutorServiceMetrics.java

```diff
@@ -0,0 +1,31 @@
+package org.springframework.metrics.instrument.scheduling;
+
+import org.springframework.metrics.instrument.MeterRegistry;
+import org.springframework.metrics.instrument.binder.MeterBinder;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+
+public class ExecutorServiceMetrics implements MeterBinder {
+
+  private final String name;
+  private final ExecutorService executorService;
+
+  public ExecutorServiceMetrics(String name, ExecutorService executorService) {
+    this.name = name;
+    this.executorService = executorService;
+  }
+
+  @Override
+  public void bindTo(MeterRegistry registry) {
+    if(executorService instanceof ThreadPoolExecutor) {
+      ThreadPoolExecutor tp = (ThreadPoolExecutor) executorService;
+      registry.gauge(name+"_scheduled_total", tp, ThreadPoolExecutor::getTaskCount); //Really should be a counter
+      registry.gauge(name+"_completed_total", tp, ThreadPoolExecutor::getCompletedTaskCount); //Really should be a counter
```

Comment:
It would not be a problem for Spectator. The semantics are very similar to Prometheus, as you can see here: https://github.com/spring-projects/spring-metrics/blob/master/src/main/java/org/springframework/metrics/instrument/spectator/SpectatorToDoubleGauge.java#L37